libportal/notification.c

```diff
@@ -72,6 +72,73 @@ ensure_action_invoked_connection (XdpPortal *portal)
                                            NULL);
 }
 
+static int
+bytes_to_memfd (const gchar  *name,
+                GBytes       *bytes,
+                GError      **error)
+{
+  g_autofd int fd = -1;
+  gconstpointer bytes_data;
+  gpointer shm;
+  gsize bytes_len;
+
+  fd = memfd_create (name, MFD_ALLOW_SEALING);
+  if (fd == -1)
+    {
+      int saved_errno;
+
+      saved_errno = errno;
```

Comment:
Nitpick:

```suggestion
      int saved_errno = errno;
```

throughout the file